doc/user/content/concepts/views.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Views (Non-Materialized and Materialized)
+description: Conceptual page on views.
+menu:
+  main:
+    parent: concepts
+    weight: 15
+    identifier: 'concepts-views'
+---
+
+## Overview
+
+Views represent queries that are saved under a name for reference. Views provide
```

Comment:
Added new content.  
There is a bit of repetition. However, in order to undo a bit of the historical emphasis on mat view, I think a little repetition can help.

Reply:
Made some comments related to how we want to approach these concepts, so we can discuss the gist in our call today before I make any suggestions.